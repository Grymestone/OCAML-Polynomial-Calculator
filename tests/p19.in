3a^2 + 8a^9 - 1a^2
