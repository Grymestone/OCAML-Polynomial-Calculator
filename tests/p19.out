(((3*(a)^2)+(8*(a)^9))-(1*(a)^2))
2x^2 + 8x^9